### PR TITLE
Dropping .py extension for artemis_cli, implemented `artemis_cli results` and minor changes

### PR DESCRIPTION
**Pull request by oberrich**
Merge `develop` -> `master`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,25 +22,26 @@ exercises in order to grade them. **This script assists the relevant steps in th
 ## First steps
 1. Add your ArTEMiS credentials (username and password) into `config.yml`
```
Comment (lgerhardt45) [inline README.md]: Add the `general` field here if needed!

File: artemis_cli [hunk 2]
```diff
@@ -235,6 +283,7 @@ def main():
         cfg = yaml.safe_load(config_file)
 
     # alias commonly used config fields
+    general = cfg['general']
```
Comment (lgerhardt45) [inline artemis_cli]: @oberrich This was not added in config.yml or the changes not pushed (probably the latter)? It should also be caught (please enter general (what does it actually do?) in config.yml) and mentioned in README, if that's an action, the user should take.

File: artemis_cli [hunk 3]
```diff
@@ -131,22 +133,28 @@ def command_repos():
...
 
-        # add student id to projectDescription/name in .project file to allow repo batch import into Eclipse
-        # tests is a Maven Project (no .project file), solution already has 'Solution' attached to name
-        if student not in ['tests', 'solution']:
-            project_file_dir = os.path.abspath(os.path.join(repo_dir, '.project'))
-            if os.path.exists(project_file_dir):
-                et = ElementTree.parse(project_file_dir)
-                et.getroot().find('name').text += ' ' + student
-                et.write(project_file_dir)
+        if general['fix_eclipse_import']:
```
Comment (lgerhardt45) [inline artemis_cli]: 👍🏼 as mentioned, `general` missing in `config.yml`

File: README.md (hunk 1 above)
Comment (oberrich) [reply README.md]: Documenting this publicly isn't necessary at this point as "fix_eclipse_imports" is set by default and most users won't care to change this.

File: artemis_cli (hunk 2 above)
Comment (oberrich) [reply artemis_cli]: Good catch, config.yml is in .gitignore in order to not accidentally leak passwords

**Pull request merged**